rpcserver/router.go

```diff
@@ -1615,6 +1615,28 @@ func (server *routedBoltzServer) GetWallet(ctx context.Context, request *boltzrp
 	return server.serializeWallet(wallet)
 }
 
+func (server *routedBoltzServer) GetWalletSendFee(ctx context.Context, request *boltzrpc.WalletSendRequest) (*boltzrpc.WalletSendFee, error) {
+	ownWallet, err := server.getOwnWallet(ctx, onchain.WalletChecker{Id: &request.Id})
+	if err != nil {
+		return nil, err
+	}
+	feeRate := request.GetSatPerVbyte()
+	if feeRate == 0 {
+		feeRate, err = server.onchain.EstimateFee(ownWallet.Currency, request.GetIsSwapAddress())
+		if err != nil {
+			return nil, err
+		}
+	}
+	if request.GetIsSwapAddress() && request.GetAddress() == "" {
```

Comment:
```suggestion
	if request.GetAddress() == "" {
```
Shouldn't we always coalesce?